src/build/emitter.ts

```diff
@@ -826,6 +826,14 @@ export function emitWebIdl(
     entity: { comment?: string; deprecated?: boolean },
     print: (s: string) => void
   ) {
+    if (
+      entity.deprecated &&
+      entity.comment?.endsWith(`*/`) &&
+      !entity.comment?.includes("@deprecated")
```

Comment:
when is this true? I thought that all the `@deprecated` tags would be inserted on line 845.

Edit: I see that the previous code also includes this check, but I think it's worth taking it out and where extra `@deprecated` are coming from, if any.

Reply:
Some entity comments have deprecated tags already included in the comment block. Some examples would be ScriptProcessorNode and PerformanceTiming. This condition is true if the entitity is deprecated, and has a multi-line comment, but does not yet contain the @deprecated tag. I will clean it up to be more readable.